lua/entities/train_fucked/shared.lua

```diff
@@ -0,0 +1,8 @@
+ENT.Type = "anim"
+ENT.Base = "base_gmodentity"
+ 
+ENT.PrintName		= "Train fucked!"
```

Comment:
Got tabs here, stick with spaces!
You should be able to configure your editor to always use spaces